src/panels/detail/components/connections/ConnectionCache.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { AppSettings, WorkspaceLinks } from "../../../../config/Variables";
+import { getLabelOrBlank } from "../../../../function/FunctionGetVars";
+import { getVocabularyShortLabel } from "@opendata-mvcr/assembly-line-shared";
+import { Badge, OverlayTrigger, Popover } from "react-bootstrap";
+import { CacheSearchVocabularies } from "../../../../datatypes/CacheSearchResults";
+import _ from "underscore";
+import { getOtherConnectionElementID } from "../../../../function/FunctionLink";
+import { Representation } from "../../../../config/Enum";
+import Connection from "./Connection";
+
+interface Props {
+  connection: {
+    link: string;
+    linkLabels: { [key: string]: string };
+    target: {
+      iri: string;
+      labels: { [key: string]: string };
+      description: { [key: string]: string };
+      vocabulary: string;
+    };
+    direction: string;
+  };
+  projectLanguage: string;
+  update: Function;
+  elemID: string;
+  selected: boolean;
+  selection: string[];
+  updateSelection: (ids: string[]) => void;
+}
+
+interface State {}
+
+export default class ConnectionCache extends React.Component<Props, State> {
+  checkVocabulary(vocabulary: string) {
```

Comment:
What is this supposed to do? It is weird that a function named "check..." returns a non-boolean value.